sql-cli/sql_cli/run_dag.py

```diff
@@ -138,7 +138,7 @@ def _run_task(ti: TaskInstance, session: Session) -> None:
 
     :param ti: TaskInstance to run
     """
-    if ti.map_index >= 0:
+    if hasattr(ti, "map_index") and ti.map_index >= 0:
```

Comment:
This is required for Airflow version prior to 2.3 which we currently allow i.e. `>2.0`.